ui/src/app/core/auth.service.ts

```diff
@@ -50,6 +63,12 @@ export class AuthService {
       if (!capabilities.authentication || !capabilities.authentication.isRequired) {
         return;
       }
+
+      if (capabilities.authentication.forcedLogoutDomains && capabilities.authentication.forcedLogoutTime && capabilities.authentication.forcedLogoutTime > (this.WARNING_INTERVAL * 2)) {
```

Comment:
I'm not sure I understand the meaning of `WARNING_INTERVAL * 2`.

Is it saying that if the `forcedLogoutTime` is really short - only twice the warning interval - don't bother showing a warning?

Reply:
This isn't a test of whether or not to show a warning, it's whether or not to set up the variables that will be used to force a logout after an idle period (see line 47).

Reply:
Why would we skip setting the variables if `capabilities.authentication.forcedLogoutTime ≤ (this.WARNING_INTERVAL * 2)`

Reply:
This was to guard against someone configuring the timeout to be an unreasonably short time and having the UI just keep logging users out.  The only validation of `capabilities.authentication.forcedLogoutTime` that's in place is that it needs to be a valid `int`.